enemy_territory/etlegacy/README.md

```diff
@@ -1,9 +1,6 @@
-# ET: Legacy
+# Wolfenstein: Enemy Territory (Legacy)
```

Comment:
This is specifically et:legacy which is the project name given by the developers. You can open their website etlegacy.com to view it